src/services/legal.services.ts

```diff
@@ -1,35 +1,22 @@
 // Libraries
 import { axios } from '@/utils'
-import { Component, Vue } from 'vue-property-decorator'
-import { Action, Getter } from 'vuex-class'
 import { NOT_FOUND } from 'http-status-codes'
-
-// Interfaces
-import {
-  ActionBindingIF, DissolutionFilingIF, IncorporationFilingIF, RegistrationFilingIF
-} from '@/interfaces'
+import { DissolutionFilingIF, IncorporationFilingIF, RegistrationFilingIF } from '@/interfaces'
 import { FilingTypes } from '@/enums'
 
 /**
- * Mixin that provides integration with the Legal API.
+ * Class that provides integration with the Legal API.
  */
-@Component({})
-export default class LegalApiMixin extends Vue {
-  @Getter getBusinessId!: string
-  @Getter getFilingId!: number
-  @Getter getTempId!: string
-
-  @Action setFilingId!: ActionBindingIF
-  @Action setFilingType!: ActionBindingIF
-
+export default class LegalServices {
```

Comment:
I changed this from a mixin to a service class. Now it doesn't need store getters!